tutorials/tour/_posts/2017-02-13-generic-classes.md

```diff
@@ -9,40 +9,49 @@ categories: tour
 num: 18
 next-page: variances
 previous-page: sequence-comprehensions
+assumed-knowledge: classes unified-types
 ---
+Generic classes are classes which take a type as a parameter. They are particularly useful for collection classes.
 
-Like in Java 5 (aka. [JDK 1.5](http://java.sun.com/j2se/1.5/)), Scala has built-in support for classes parameterized with types. Such generic classes are particularly useful for the development of collection classes.
-Here is an example which demonstrates this:
-
+## Defining a generic class
+Generic classes take a type as a parameter within square brackets `[]`. One convention is to use the letter `T` as type parameter identifier, though this is not required.
```

Comment:
@SethTisue - `A` is ubiquitous in the standard library and in various other locations (IIRC it was the convention used in Programming in Scala, and it is the convention used in Functional Programming in Scala (which I can check despite being away from books because a chapter is available online).